scss/components/side-nav.scss

```diff
@@ -124,12 +124,31 @@ $block: #{$fd-namespace}-side-nav;
 
     &__icon {
         padding-right: $fd-side-nav-icon-right-padding;
+        margin-top: -1px;
+
+        &.sap-icon--s {
+            margin-top: 0;
+        }
+
+        &.sap-icon--m {
+            margin-top: 0;
+        }
+
```

Comment:
I don't think you need small or medium blocks since the value is just 0.  I don't think icons have a top margin by default.

Reply:
there is one more case that is sap-icon w/o any size indicated, and it is  margin-top: -1px, so if i don't give small or medium blocks, it will take margin-top: -1px.